flow/scripts/escape.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+sed -e 's/ /\\ /g' -e 's/(/\\(/g' -e 's/|/\\|/g' -e 's/)/\\)/g'
```

Comment:
Do [].*^$\ need escaping?

Reply:
Yes, but I fought a bit with make, bash and sed w.r.t. escaping hell and lost...

I don't see these other chars used by ORFS variables.

Can we merge this? It is a step forward...

To test further escape rules, run something like:

```
$ make  "GDS_ALLOW_EMPTY=(a|b)" test-unset-and-make-print-GDS_ALLOW_EMPTY
GDS_ALLOW_EMPTY = (a|b)
```